EventEnabledInsurance/QuoteLifecycleSimulator/deploy-simulator.sh

```diff
@@ -38,20 +38,14 @@ PG_PORT=5432
 TICK_MILLIS=1000
 MOBILE_TEST_ROWS=10
 
-while getopts "n:t:m:" opt; do
+while getopts "n:" opt; do
   case ${opt} in
   n)
     namespace="$OPTARG"
     ;;
-  t)
-    TICK_MILLIS="$OPTARG"
-    ;;
-  m)
-    MOBILE_TEST_ROWS="$OPTARG"
-    ;;
   \?)
     usage
-    exit
+    # exit
```

Comment:
Any reason this is commented out?

Reply:
The usage already has an exit inside it, so removed this exit.